.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
we could also update this repo to use main instead of master 😊

Reply:
Totally agree. I don't have the access myself so I left it for now. I'll ping ops to see if they can help us out.

Reply:
I think Oz can also do it 😊

Reply:
@ozeias Could you flip the default branch to main for us?

Reply:
good call @elainenaomi, the update is done.